pkg/curl/curl_config.h

```diff
@@ -234,26 +239,25 @@
 #define HAVE_UTIME_H 1
 #define HAVE_VARIADIC_MACROS_C99 1
 #define HAVE_VARIADIC_MACROS_GCC 1
-/* #undef HAVE_WINCRYPT_H */
 /* #undef HAVE_WINDOWS_H */
-/* #undef HAVE_WINLDAP_H */
 /* #undef HAVE_WINSOCK2_H */
 /* #undef HAVE_WOLFSSH_SSH_H */
 /* #undef HAVE_WOLFSSL_DES_ECB_ENCRYPT */
+/* #undef HAVE_WOLFSSL_FULL_BIO */
 /* #undef HAVE_WOLFSSL_GET_PEER_CERTIFICATE */
 /* #undef HAVE_WOLFSSL_USEALPN */
 #define HAVE_WRITABLE_ARGV 1
 /* #undef HAVE_WS2TCPIP_H */
 /* #undef HAVE_X509_H */
-#define HAVE_ZLIB_H 1
 /* #undef HAVE_ZSTD */
 /* #undef HAVE_ZSTD_H */
+/* #undef HAVE__FSEEKI64 */
 #define LT_OBJDIR ".libs/"
 /* #undef NEED_LBER_H */
 /* #undef NEED_REENTRANT */
 /* #undef NEED_THREAD_SAFE */
-#define NTLM_WB_ENABLED 1
-#define NTLM_WB_FILE "/usr/bin/ntlm_auth"
+/* #undef NTLM_WB_ENABLED */
+/* #undef NTLM_WB_FILE */
```

Comment:
This is some binary related to samba, I don't think this should really be enabled.

Reply:
Agreed.